tests/integration/exec.bats

```diff
@@ -3,6 +3,7 @@
 load helpers
 
 function setup() {
+	[ $EUID -eq 0 ] && prlimit --nofile=1024:1048576 -p $$
```

Comment:
This is ugly and although I've mentioned it (for completeness), I don't recommend doing it, as it is a global setting which affects all the tests down the line.

Easier and cleaner to set lower limits in the tests.

Reply:
But in some environments, the soft and hard value of nofile limit are equal. I moved all these type tests to `rlimits.bat`.

Reply:
I think I was wrong about the setting being global. Every test case is executed under a separate shell. So it's all right.